cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -186,11 +186,12 @@ func TestStaticAutoscalerRunOnce(t *testing.T) {
 			ScaleDownUnreadyTime:          time.Minute,
 			ScaleDownUtilizationThreshold: 0.5,
 		},
-		EstimatorName:    estimator.BinpackingEstimatorName,
-		ScaleDownEnabled: true,
-		MaxNodesTotal:    1,
-		MaxCoresTotal:    10,
-		MaxMemoryTotal:   100000,
+		EstimatorName:                    estimator.BinpackingEstimatorName,
+		ScaleUpToNodeGroupMinSizeEnabled: true,
```

Comment:
Why set the flag to `true` here? This isn't actually testing the new logic. I think there should be a separate test case validating whether new scale up logic to enforce min size got triggered. Ideally, the tests here should be refactored to be table-based (in which case the new test would just become one extra test case on the list), but that requires a bit more refactoring.

Reply:
My intension was to ensure the new flag doesn't block the existing test, since most of the scale up logic has been tested in `scale_up_test.go`. Adding a new test case also sounds reasonable to me, will do it in the next iteration.